.travis.yml

```diff
@@ -2,9 +2,9 @@ language: java
 os:
   - linux
 jdk:
-  - openjdk8
+  - openjdk11
```

Comment:
I think we should add a job below using `openjdk8`, that way we know that we are not breaking java 8 support.